repository_test/repository/pm-be0c7738-039d-9445-8237-8b85f37cd303-remote/data.json

```diff
@@ -77,6 +77,12 @@
 			"url": "be0c7738-039d-9445-8237-8b85f37cd303.es.dfxp"
 		},
 		{
```

Comment:
Create a new repo with the VTT example, don't reuse the `pm-be0c7738-039d-9445-8237-8b85f37cd303-remote`.